src/features/surveys/components/SurveySubmissionsList.tsx

```diff
@@ -216,6 +218,7 @@ const SurveySubmissionsList = ({
     return (
       <ZUIPersonGridEditCell
         cell={row.respondent}
+        onCreate={() => setCreatePersonOpen(true)}
```

Comment:
What if instead of a boolean state, we use a state for the ID of the submission that triggered it, and pass it from the edit cell via `onCreate()`. That could allow for linking the person after `ZUICreatePerson` has finished creating it.